File: leetcode/Length_of_Last_Word.java
import java.util.*;

public class Solution {
    public int lengthOfLastWord(String s) {
        s = s.trim();
        int o = s.lastIndexOf(' ');
        if (o == -1) return s.length();
        return s.length() - 1 - o;       
    }
}

File: leetcode/Length_of_Last_Word.cpp
class Solution {
public:
    int lengthOfLastWord(const char *s) {
        int length = strlen(s);
        int last_word_length = 0;
        int count = 0;
        for (int i = 0; i <= length; ++i)
            if (i == length || s[i] == ' ') {
                if (count > 0)
                    last_word_length = count;
                count = 0;
            } else {
                ++count;
            }
        return last_word_length;
    }
};
